projects/crossplane/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/crossplane/crossplane"
+main_repo: "https://github.com/crossplane/crossplane"
+primary_contact: ""
```

Comment:
looks like @muvaf and @negz have given the go ahead to use their email address as maintainers for crossplane in this file - this initial config seems good to me after that!